### PR TITLE
add cerberus schema

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: libcflib/model.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import json
```
Comment (scopatz) [inline libcflib/model.py]: Please add a docstring

File: libcflib/model.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import os
+
```
Comment (scopatz) [inline libcflib/model.py]: No need to blank line between standard lib imports

File: libcflib/model.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+from collections.abc import MutableMapping
+from typing import Iterator
+
+
+class Node(MutableMapping):
```
Comment (scopatz) [inline libcflib/model.py]: Pleas add a docstring

File: libcflib/schemas.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+schemas = {
```
Comment (scopatz) [inline libcflib/schemas.py]: PEP8 global variables should be ALL_UPPERCASE

File: libcflib/model.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        if k not in self.EXCLUDE_LOAD:
+            self._load()
+        del self.__dict__[k]
+
+    def __setitem__(self, k, v) -> None:
+        if k not in self.EXCLUDE_LOAD:
+            self._load()
+        self.__dict__[k] = v
+
+    def __init__(self, name, folder):
```
Comment (scopatz) [inline libcflib/model.py]: init is usually the first method. weird to see it way down here

**Comment by CJ-Wright** [comment]
Add `bad`, `harvest_time`.

**Pull request merged**